commit f6c069cfb438086206ef34206e322e8dab18973a (refs/changes/06/1306/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-01-20T14:59:59-08:00 (2 years, 9 months ago)
    
    IMGAPI-612 IMGAPI suffering from IMGAPI-599 which breaks http_proxy support
    Reviewed by: Julien Gilli <julien.gilli@joyent.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/package.json b/package.json
index 6670eed..7596fb1 100644
--- a/package.json
+++ b/package.json
@@ -34,7 +34,7 @@
     "progbar": "git+https://github.com/trentm/node-progbar.git#a4c56e6",
     "restify": "4.1.1",
     "rimraf": "2.2.6",
-    "sdc-clients": "10.0.1",
+    "sdc-clients": "10.0.4",
     "semver": "3.0.1",
     "sshpk": "1.10.0",
     "trace-event": "1.2.0",
